src/LinAlg/hiopMatrixRajaDense.cpp

```diff
@@ -1365,7 +1365,7 @@ double hiopMatrixRajaDense::max_abs_value()
   RAJA::forall<hiop_raja_exec>(RAJA::RangeSegment(0, n_local_ * m_local_),
     RAJA_LAMBDA(RAJA::Index_type i)
     {
-      norm.max(fabs(dd[i]));
+      norm.max(std::fabs(dd[i]));
```

Comment:
I am not sure if this is appropriate implementation. I am not sure if CUDA, for example, supports `std` math functions.